packages/cluster/src/Entity.ts

```diff
@@ -0,0 +1,137 @@
+/**
+ * @since 1.0.0
+ */
+import type { Schema, TaggedRequest } from "@effect/schema/Schema"
+import type * as Serializable from "@effect/schema/Serializable"
+import * as Data from "effect/Data"
+import * as Equal from "effect/Equal"
+import * as Hash from "effect/Hash"
+import * as Predicate from "effect/Predicate"
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export const TypeId: unique symbol = Symbol.for("@effect/cluster/Entity")
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export type TypeId = typeof TypeId
+
+/**
+ * An `Entity` represents a logical unit for processing work within the cluster
+ * that is capable of maintaining its own state and processing incoming
+ * messages. The behavior of an `Entity` is defined by the set of operations or
+ * actions it can perform in response to an incoming message.
+ *
+ * An `Entity` can either be a `Standard` entity, which represents an entity
+ * that is only ever alive on a single pod in the cluster at a time, or a
+ * `Clustered` entity, which can be alive on multiple pods in the cluster at a
+ * time.
+ *
+ * @since 1.0.0
+ * @category models
+ */
+export type Entity<Msg extends TaggedRequest.Any> =
```

Comment:
Given the description of a `RecipientType`:

"A RecipientType is basically a pointer to a logical grouping of multiple entities"

I don't think `Entity` is the right name for this refactor.

Maybe something like:

```ts
type EntityGroup = Single | Multiple
```